[2023-02-08T15:17:08.184+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: cms_asp_noc.populate_all_tables scheduled__2022-10-05T00:00:00+00:00 [queued]>
[2023-02-08T15:17:08.208+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: cms_asp_noc.populate_all_tables scheduled__2022-10-05T00:00:00+00:00 [queued]>
[2023-02-08T15:17:08.209+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-08T15:17:08.210+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-08T15:17:08.210+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-08T15:17:08.235+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): populate_all_tables> on 2022-10-05 00:00:00+00:00
[2023-02-08T15:17:08.244+0000] {standard_task_runner.py:55} INFO - Started process 2834 to run task
[2023-02-08T15:17:08.253+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cms_asp_noc', 'populate_all_tables', 'scheduled__2022-10-05T00:00:00+00:00', '--job-id', '973', '--raw', '--subdir', 'DAGS_FOLDER/cms_upd.py', '--cfg-path', '/tmp/tmpmpjpyh3l']
[2023-02-08T15:17:08.257+0000] {standard_task_runner.py:83} INFO - Job 973: Subtask populate_all_tables
[2023-02-08T15:17:08.397+0000] {task_command.py:388} INFO - Running <TaskInstance: cms_asp_noc.populate_all_tables scheduled__2022-10-05T00:00:00+00:00 [running]> on host 1f90076947be
[2023-02-08T15:17:08.910+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cms_asp_noc
AIRFLOW_CTX_TASK_ID=populate_all_tables
AIRFLOW_CTX_EXECUTION_DATE=2022-10-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-05T00:00:00+00:00
[2023-02-08T15:17:08.912+0000] {sql.py:254} INFO - Executing: ['call sp_upsert_asp_pricing();\ncall sp_upsert_asp_ndc_hcpcs();\ncall sp_upsert_noc_NDC_HCPCS();']
[2023-02-08T15:17:08.937+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 255, in execute
    hook = self.get_db_hook()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 179, in get_db_hook
    return self._hook
  File "/home/airflow/.local/lib/python3.7/site-packages/cached_property.py", line 36, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 141, in _hook
    conn = BaseHook.get_connection(self.conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/connection.py", line 435, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `postgres_conn` isn't defined
[2023-02-08T15:17:08.941+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=cms_asp_noc, task_id=populate_all_tables, execution_date=20221005T000000, start_date=20230208T151708, end_date=20230208T151708
[2023-02-08T15:17:08.952+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 973 for task populate_all_tables (The conn_id `postgres_conn` isn't defined; 2834)
[2023-02-08T15:17:08.997+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-08T15:17:09.042+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
